From 48b1fe94844a03fd76f903b5191232fc5e786f9e Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Fri, 8 Jan 2016 01:12:46 +0900
Subject: [PATCH 2108/2848] YARN-4544. All the log messages about rolling
 monitoring interval are shown with WARN level.
 Contributed by Takashi Ohnishi.

(cherry picked from commit 52b77577c44538c26fb9d0144f0b280be3593a38)
(cherry picked from commit 3905435d3d592b980425169bf1355d3d2b5c616f)

Change-Id: I493d60f9bcb451e54997259f12da3e630cb39ad8
---
 .../logaggregation/AppLogAggregatorImpl.java       |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java
index a679140..bf6346b 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/AppLogAggregatorImpl.java
@@ -187,12 +187,12 @@ public AppLogAggregatorImpl(Dispatcher dispatcher,
       }
     } else {
       if (configuredRollingMonitorInterval <= 0) {
-        LOG.warn("rollingMonitorInterval is set as "
+        LOG.info("rollingMonitorInterval is set as "
             + configuredRollingMonitorInterval + ". "
-            + "The log rolling mornitoring interval is disabled. "
+            + "The log rolling monitoring interval is disabled. "
             + "The logs will be aggregated after this application is finished.");
       } else {
-        LOG.warn("rollingMonitorInterval is set as "
+        LOG.info("rollingMonitorInterval is set as "
             + configuredRollingMonitorInterval + ". "
             + "The logs will be aggregated every "
             + configuredRollingMonitorInterval + " seconds");
-- 
1.7.9.5

